demo/full/scripts/controllers/knobs/VideoTrack.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import withModulesState from "../../lib/withModulesState.jsx";
+import Knob from "../../components/Knob.jsx";
+
+const findVideoTrackIndex = (trackId, tracks) => {
+  return tracks.findIndex(tk => tk.id === trackId);
+};
+
+const VideoTrackKnobBase = ({
+  player,
+  currentVideoTrackId,
+  availableVideoTracks = [],
+}) => {
+    debugger;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Remove this debugger statement. ([javascript:DebuggerStatement](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3ADebuggerStatement))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=278&issues=AWUjBS2lzFyjFVlut9x2&open=AWUjBS2lzFyjFVlut9x2)

[ISSUE_KEY]: AWUjBS2lzFyjFVlut9x2